app/views/incident/transportMeans/IdentificationNumberView.scala.html

```diff
@@ -42,9 +43,10 @@
 
         @inputText(
             field = form("value"),
-            inputTextType = OrdinaryTextInput(
+            inputTextType = TextInputWithHiddenLabel(
                 heading = messages("incident.transportMeans.identificationNumber.heading", identification.arg),
```

Comment:
Looks like we're passing args in where they're now not needed. Same for the title as well?